airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/spec.json

```diff
@@ -4,49 +4,9 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Google Analytics V4 Spec",
     "type": "object",
-    "required": ["credentials", "view_id", "start_date"],
-    "additionalProperties": true,
+    "required": ["authorization", "view_id", "start_date"],
```

Comment:
we should keep the name the same since [other code](https://github.com/airbytehq/airbyte/blob/master/airbyte-oauth/src/main/java/io/airbyte/oauth/flows/google/GoogleAnalyticsOAuthFlow.java#L57) in the platform depends on this